core/src/components/tab-button/tab-button.ionic.scss

```diff
@@ -12,8 +12,12 @@
 
   position: relative;
 
+  flex: 1;
+
```

Comment:
```suggestion
```

This isn't necessary since it is defined in [tab-button.common.scss](https://github.com/ionic-team/ionic-framework/blob/cac12b31fbf2d6da49250c12e8933c87e10fd435/core/src/components/tab-button/tab-button.common.scss#L21)